code/modules/medical/computer/cloning.dm

```diff
@@ -16,6 +16,7 @@
 	var/datum/dna2/record/active_record = null
 	var/obj/item/weapon/disk/data/diskette = null //Mostly so the geneticist can steal everything.
 	var/loading = 0 // Nice loading text
+	var/available_species = list("human","tajaran","skrell","unathi","grey","plasmamen","vox")
```

Comment:
@Arthurdentist :+1:

okay lemme clarify what I meant by this:

at /tg/ we added some shit for dangerous existence species, we use it for shit that players shouldn't be able to be without a very specific instance occuring
that way we can add a new species for code purposes without having to update a shitton of whitelists and do it from a single var

i'm goofball after all but I feel that might be a bit more useful than maintaining 50k whitelists